collectors/githubreceiver/colconfig.yaml

```diff
@@ -0,0 +1,94 @@
+- op: add
+  path: /spec/config
+  value:
+    extensions:
+      health_check: {}
+
+      pprof:
+        endpoint: 0.0.0.0:1777
+
+      zpages:
+        endpoint: 0.0.0.0:55679
+
+      bearertokenauth/github:
+        token: ${env:GH_PAT}
+
+      # bearertokenauth/gitlab:
+      #   token: ${env:GL_PAT}
+
+    receivers:
+      github:
+        initial_delay: 10s
+        collection_interval: 60s
+        scrapers:
+          github:
+            metrics:
+              #Optional Metrics
+              vcs.repository.contributor.count:
+                enabled: true
+            github_org: liatrio
+            #search_query: org:liatrio topic:o11y archived:false  #Recommended optional query override, defaults to "{org,user}:<github_org>"
+            #endpoint: https://selfmanagedenterpriseserver.com
+            auth:
+              authenticator: bearertokenauth/github
+
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 15
+
+      batch:
+        send_batch_size: 100
+        timeout: 10s
+
+      resource/o11y:
+        attributes:
+          - key: team.name
+            # Change this value if you want to associate your scraped repositories
+            # with a different team name
+            value: tag-o11y
+            action: upsert
+
+    exporters:
+      debug:
+        verbosity: normal
+        sampling_initial: 2
+        sampling_thereafter: 500
+
+      otlp:
+        endpoint: http://gateway-collector.collector.svc.cluster.local:4317
+        tls:
+          insecure: true
+
+    service:
+      extensions:
+        - health_check
+        - pprof
+        - zpages
+        - bearertokenauth/github
+      pipelines:
+        metrics:
+          receivers:
+            - github
+          processors:
+            - memory_limiter
+            - batch
+            - resource/o11y
+          exporters:
+            - debug
+            - otlp
+
+- op: add
+  path: /spec/env
+  value: 
+    - name: GH_PAT
+      valueFrom:
+        secretKeyRef:
+          name: git-pat
```

Comment:
Should we change this secret to github-pat? since this will be for the github receiver and to differentiate whenever gitlab receiver is set up for quick start